browser/src/control/Control.AutoFillOptions.ts

```diff
@@ -76,20 +75,20 @@ class AutoFillOptions extends L.Control.AutoCompletePopup {
 		data.posx = framePos.x;
 		data.posy = framePos.y;
 		this.sendJSON(data);
+
+		this.map._docLayer.isAutoFillPopupOpen = true;
```

Comment:
Please do not move implementation details outside of the class. We can store that information here - or not store but just check.

Please add `isOpen()` method to the base class, which can be used instead maybe?

Reply:
this is removed in the new commits: no need x 4